src/System.Management.Automation/engine/SessionStateStrings.cs

```diff
@@ -135,6 +135,11 @@ internal static class StringLiterals
         /// </summary>
         internal const string PowerShellNgenAssemblyExtension = ".ni.dll";
 
+        /// <summary>
+        /// The file extension (including the dot) of an executable file.
+        /// </summary>
+        internal const string PowerShellILExecutableExtension = ".exe";
```

Comment:
This will only work on Windows, right?  Or requires managed executables on non-Windows to have .exe extension.